vaporetto/src/predictor.rs

```diff
@@ -136,30 +139,7 @@ impl Predictor {
     }
 
     fn add_type_ngram_scores(&self, sentence: &Sentence, start: usize, ys: &mut [ScoreValue]) {
```

Comment:
This function is no longer necessary.
Could you call `type_predictor.add_scores()` directly?